driver-core/src/main/com/mongodb/MongoCredential.java

```diff
@@ -179,6 +183,31 @@ public final class MongoCredential {
     @Beta(Beta.Reason.CLIENT)
     public static final String AWS_CREDENTIAL_PROVIDER_KEY = "AWS_CREDENTIAL_PROVIDER";
 
+    /**
+     * The provider name. The value must be a string.
+     * <p>
+     * If this is provided,
+     * {@link MongoCredential#OIDC_CALLBACK_KEY}
+     * must not be provided.
+     *
+     * @see #createOidcCredential(String)
+     * @since 4.10
```

Comment:
Update `@since` on all new program elements, so that it points to the correct version.

Reply:
Should be done.